newton/sim/builder.py

```diff
@@ -418,6 +418,19 @@ def __init__(self, up_axis: AxisType = Axis.Z, gravity: float = -9.81):
         # if setting is None, the number of worst-case number of contacts will be calculated in self.finalize()
         self.num_rigid_contacts_per_env = None
 
+        # equality constraints
+        self.equality_constraint_type = []
+        self.equality_constraint_body1 = []
+        self.equality_constraint_body2 = []
+        self.equality_constraint_anchor1 = []
+        self.equality_constraint_anchor2 = []
+        self.equality_constraint_distance = []
+        self.equality_constraint_joint1 = []
+        self.equality_constraint_joint2 = []
+        self.equality_constraint_polycoef = []
+        self.equality_constraint_key = []
+        self.equality_constraint_enabled = []
```

Comment:
It would be good to document these member variables, like we do for the other attributes.